ls1intum/Artemis
# `Text exercises`: Fix a type error for empty text blocks

### Checklist
#### General
#### Client

### Motivation and Context
A student wrote [on Zulip](https://zulip.in.tum.de/#narrow/stream/934-EIST22-Organizational/topic/Artemis.20Error/near/590055) that he was unable to see his feedback. In the console a "cannot read properties from undefined" error was show.

### Description
After looking at the network log, for some reason the text-attribute was undefined:
![grafik](https://user-images.githubusercontent.com/26540346/170711611-cdfa2cc3-4032-4a26-b296-5c157f7887e8.png)

In this case, we'll now use an empty string as the content of this block.

### Steps for Testing
Thanks to @bottbenj for the test instructions:

The submission needs to have (multiple?) empty lines at the end and then in assessment you need to select the very last line by clicking it twice with alt pressed. You can then attach a feedback to an empty text block which when sending to the server will still correctly have an empty string as its text property, but when the server sends it back to the student the text will be gone.

After that, go to the student's view and make sure that they can see the result.

### Review Progress





#### Code Review
#### Manual Tests
